pkgs/by-name/in/industrializer/package.nix

```diff
@@ -48,8 +50,14 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   strictDeps = true;
-
-  preConfigure = "./autogen.sh";
+  
```

Comment:
```suggestion

```
very suttle, thanks CI!

Reply:
I ran ``nixfmt``, and the diff indicated something changed, but I'm still not sure what.
A space got removed, I guess?
Very subtle indeed!